accel/sel4/sel4-all.c

```diff
@@ -348,6 +389,14 @@ static void sel4_region_del(MemoryListener *listener, MemoryRegionSection *secti
     tii_printf("WARNING: %s entered, but no real implementation\n", __func__);
 }
 
+static void s2_fault_doorbell(void *cookie)
+{
+    uint32_t *event_bar = cookie;
+
+    /* data does not matter, just the write fault */
+    event_bar[0] = 1;
```

Comment:
A write barrier (or write release) after this would be good to ensure ordering of writes